src/models/types.ts

```diff
@@ -1,10 +1,6 @@
 export type ResolverServices = "redefined" | "ens" | "unstoppable"
 
-export type NodeNetwork = "eth" | "arbitrum" | "bsc"| "zil";
```

Comment:
I dropped all networks
Because in the future we will use networks received on request